cmd/exec/exec.go

```diff
@@ -62,7 +62,7 @@ func CreateCommand(parent *prm.Prm) *cobra.Command {
 	err = viper.BindPFlag("toolpath", tmp.Flags().Lookup("toolpath"))
 	cobra.CheckErr(err)
 
-	tmp.Flags().StringVar(&prmApi.CodeDir, "codedir", "", "location of code to execute against")
+	tmp.Flags().StringVar(&prmApi.CodeDir, "codedir", ".", "location of code to execute against")
```

Comment:
I wonder if we should be more explicit here and use `os.Getwd()`? 

It adds a few more lines with you tack on the error check but it removes any doubt about paths as you will always get the absolute path of the current working directory.